docs/topics/azure-api-throttling.md

```diff
@@ -0,0 +1,532 @@
+# Azure API Throttling
+
+## Overview
+
+Azure has hard limits on the number of read and write requests against Azure APIs *per subscription*. Running lots of clusters in a single subscription, or running a single large, dynamic cluster in a subscription can produce side effects that exceed the number of calls permitted within a given time window for a particular category of requests. See the following documents for more detail on Azure API throttling in general:
+
+- https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/request-limits-and-throttling
+- https://docs.microsoft.com/en-us/azure/virtual-machines/troubleshooting/troubleshooting-throttling-errors
+
+## Running older versions of the Azure cloudprovider runtime
+
+Over time, the Azure cloudprovider runtime has optimized its behaviors to reconcile Azure resource requests (network, compute, storage) with a minimum number of calls to the Azure APIs in order to prevent Azure API throttling. In practice, running versions of Kubernetes prior to the below list has known issues with generating excess Azure API requests for clusters with VMSS node pools:
+
+- 1.15.12
+- // TODO 1.16 version

```

Comment:
@aramase do we have definitive 1.16 and 1.17 versions to recommend folks use w/ the bulk of cloudprovider improvements related to Azure API throttling?

Reply:
`1.17.3` has the required cache improvements for reading from stale cache, fix throttling during scale down and LB fix. 
`1.16.7` is the minimum version that has the same cache improvements

But I would recommend at least `1.17.5` and `1.16.9` minimum because these versions also contains fixes for disks (dangling attach issue)